### PR TITLE
README: add a community section

### PR DESCRIPTION
**Pull request by dongsupark**
Add a section of community, including our slack channel on https://kubernetes.slack.com.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -175,3 +175,7 @@ sudo make install
 ## Troubleshooting
 
 See [doc/troubleshooting](doc/troubleshooting.md)
+
+## Community
+
+* Slack: #kube-spawn on https://kubernetes.slack.com
```
Comment (blixtra) [inline README.md]: I think you can add a link to the actual channel, which would be more helpful.

**Comment by dongsupark** [comment]
Fixed.

File: README.md [hunk 2]
```diff
@@ -175,3 +175,7 @@ sudo make install
 ## Troubleshooting
 
 See [doc/troubleshooting](doc/troubleshooting.md)
+
+## Community
+
+* Slack: #kube-spawn on https://kubernetes.slack.com/messages/C9ZMJH2NL/
```
Comment (blixtra) [inline README.md]: How about simply `Discuss the project on [Slack](https://kubernetes.slack.com/messages/C9ZMJH2NL/)`?

Comment (dongsupark) [reply README.md]: That's a good idea. Fixed.

**Pull request merged**